blackbox/docs/sql/scalar.txt

```diff
@@ -215,6 +215,41 @@ locale-sensitive or context-sensitive mappings.
     SELECT 1 row in set (... sec)
 
 
+sha1('string') returns string
+-----------------------------
+
+Computes the SHA1 checksum of the given string.
+
+::
+
+    cr> select sha1('crate')
+    ... limit 1;
+    +------------------------------------------+
+    | sha1('crate')                            |
+    +------------------------------------------+
+    | 1673dc397042322a0a5ac49c79cc08d3a25cb0f6 |
+    +------------------------------------------+
+    SELECT 1 row in set (... sec)
+
+sha256('string') returns string
```

Comment:
You're right, internals APIs should not dictate what is possible for the user. So let's revisit the SHA2 hash functions interface discussion at a later point in time and keep only SHA1 and MD5 for now. Perhaps users will provide some feedback on how the interface should look like when they start using the new hash functions.